src/backend/cod3.c

```diff
@@ -3268,22 +3267,15 @@ code* prolog_genvarargs(symbol* sv, regm_t* namedargs)
         }
     }
 
-    bool saveRDX = (mask[DX] & *namedargs) != 0;
-    if (saveRDX)
-    {
-        c = genregs(c,0x89,DX,R11); // MOV R11,RDX
-        code_orrex(c, REX_W);
-    }
-
     c = genregs(c,0x0FB6,AX,AX);                          // MOVZX EAX,AL
     genc2(c,0xC1,modregrm(3,4,AX),2);                      // SHL EAX,2
     int raxoff = voff+6*8+0x7F;
-    unsigned L2offset = (raxoff < -0x7F) ? 0x2C : 0x29;
+    unsigned L2offset = (raxoff < -0x7F) ? 0x2D : 0x2A;
```

Comment:
The tricky path was the `jmp R11d` adds one byte for the REX prefix.
It's pretty ugly to have the complete jump offset in one number instead of adding it up for each instruction individually.